src/components/DisplayComponent.js

```diff
@@ -113,6 +121,24 @@ class DisplayComponent extends Component {
     }
   }
 
+  handleChangeSearch = event => {
+    let search = event.target.value.toLowerCase();
+    search = search.replace('\\', '');
+    search = search.replace('*', '');
+    this.setState(
+      {
+        searchBoxText: search
+      },
+      this.apiCall
```

Comment:
I dont see this method anywhere, can this be removed?

Reply:
yes.